tests/message/test_messages.py

```diff
@@ -345,3 +345,20 @@ def test_message_remove_task_container(self):
             )
         for i in range(test_cases):
             self.assertEqual(msg_l.remove_tasks[i].task_id, task_ids[i])
+
+    def test_message_task_to_compute_with_luxrender_script_package(self):
+        compute_task_def = factories.tasks.ComputeTaskDefFactory.with_luxrender_script_package()
+        task_to_compute = factories.tasks.TaskToComputeFactory(compute_task_def=compute_task_def)
+        serialized_message = shortcuts.dump(task_to_compute, None, None)
+        deserialzied_message = shortcuts.load(serialized_message, None, None)
+        self.assertEqual(task_to_compute, deserialzied_message)
+
+    def test_message_task_to_compute_with_blender_script_package(self):
```

Comment:
IMO this should be moved to `tests/message/te_tasks.py`

Reply:
Oki, moved.